app/views/adopter_profiles/show.html.erb

```diff
@@ -90,7 +90,7 @@
           </div>
           <div class="row mb-4 bigger">
             <p class="col-lg-4">
-              <strong><%= t('.hours_alone_on_weekdays') %></strong>
+              <strong>Hours pet will be alone on weekdays</strong>
```

Comment:
This can be left as `t('.hours_alone_on_weekdays')` - we just want to update the corresponding text in `en.yml`

Reply:
Silly me and thank you